docs/workforce-management-framework/rhmap/using-wfm-templates-on-rhmap.adoc

```diff
@@ -0,0 +1,46 @@
+= Using {WFM-RC-nameLong} in RHMAP
+
+{WFM-RC-nameLong} can be deployed using the templates built into RHMAP.
+
+== Start a new Project
+
+- 1. In the platform select `Projects` in the top navigation bar
+- 2. In the Projects screen select `New Project`
+- 3. In the `Choose a Template` screen select the `WFM Project` by clicking `Choose`
+- 4. Name your Project and click on the `Create`
+
+NOTE: You can apply different names to the individual appication within the Project
+
+- 5. Once the Project is created click `Finish` and the Project will render with the three application
+
+== Setting up the WFM Cloud App
+
+- 1. Select the `WFM Cloud App` 
+- 2. If the database needs updating select `Data Browser` on the left navigation bar
+- 3. Click `Upgrade database`
+- 4. You will recive a warning, click on `Upgrade now` to proceed 
+- 5. Once the Upgrade has completed click on `Next`
+- 6. On the next screen click on `Deploy`
+- 7. On the Deploy screen in the `Choose a Runtime` dropdown select `Node.js - 6.1.9`
+- 8. Click on the `Deploy Cloud App`
+- 9· Once deployed return to the Project screen by clicking on `Apps, Cloud Apps & Services`
+
+NOTE: New changes to the WFM Cloud App code base to be deployed
+
+== Setting up the WFM Portal App
+
+- 1. Select the `WFM Portal App` 
+- 2. To deploy the Portal appliction select `Deploy` on the left navigation panel
+- 3. On the Deploy screen in the `Choose a Runtime` dropdown select `Node.js - 6.1.9`
+- 4. Click on the `Deploy Portal App`
+- 5· Once deployed return to the Project screen by clicking on `Apps, Cloud Apps & Services`
+
+NOTE: New changes to the WFM Portal App code base to be deployed
```

Comment:
What you mean by this: `New changes to the WFM Portal App code base to be deployed`

Reply:
Let's use: "Everytime you change content of the app your application needs to be deployed"